functions.php

```diff
@@ -476,3 +476,50 @@ function ( $downsize, $id, $size ) {
 	100,
 	3
 );
+
+// This action overrides the WordPress functionality for adding a notice message
+// https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/edit-form-blocks.php#L303-L305
+// When it's a page for posts.
+add_action(
+	'admin_enqueue_scripts',
+	function() {
+		global $post;
+
+		if ( $post && (int) get_option( 'page_for_posts' ) === $post->ID ) {
+			// Adding this style, it works as a workaround for editors
+			// To disable the ability to edit the content of the listing page.
+			echo '<style>
+			.edit-post-header-toolbar,
+			.block-list-appender {
+				pointer-events: none;
+				opacity: 0;
+			}
+
+			.block-editor-block-list__layout {
+				display: none;
+			}
+		</style>';
+
+			wp_add_inline_script(
+				'wp-notices',
+				sprintf(
+					'wp.data.dispatch( "core/notices" ).createWarningNotice( "%s", { isDismissible: false } )',
+					__( 'Custom P4 message', 'planet4-master-theme' )
+				),
+				'after'
+			);
+		}
+	},
+	100
+);
+
+// The new `pagenum` query_var is used ONLY trough the default listing pages (index.php),
+// when the Load more feature is enabled.
+// Also added in replacement of the `page` query_var since that param is returning always zero.
+add_filter(
```

Comment:
Need to add this new query var since `page` is not working through the default listing pages.

Also using using `_` for a WP's convention.

Reply:
Bringing more information:

After clicking the Read More button at the first time, you'll get 2 different responses, the first one is returns `301` and the next returns `201`.

You can try also by running for example this command: 

`curl -i https://www-dev.greenpeace.org/test-tavros/news-stories\?page\=2\&page_num\=2`

And returns
```
HTTP/2 301 
date: Fri, 09 Sep 2022 15:35:00 GMT
content-type: text/html; charset=UTF-8
location: https://www-dev.greenpeace.org/test-tavros/news-stories/?page_num=2
cf-ray: 7480fdbfb8f308ed-EZE
cache-control: max-age=3600
expires: Fri, 09 Sep 2022 16:35:00 GMT
strict-transport-security: max-age=15724800; includeSubDomains
cf-cache-status: BYPASS
cf-apo-via: origin,qs
cf-edge-cache: cache,platform=wordpress
content-security-policy: default-src * 'self' data: 'unsafe-inline' 'unsafe-eval'; frame-ancestors 'self'
permissions-policy: geolocation=(),sync-xhr=(),microphone=(),camera=(),fullscreen=(),payment=()
referrer-policy: strict-origin-when-cross-origin
x-cache: BYPASS
x-cache-2: BYPASS
x-content-type-options: nosniff
x-frame-options: SAMEORIGIN
x-redirect-by: WordPress
x-xss-protection: 1; mode=block
server: cloudflare
alt-svc: h3=":443"; ma=86400, h3-29=":443"; ma=86400
```